guides/common/modules/con_boot-disk-types.adoc

```diff
@@ -0,0 +1,50 @@
+[id="boot-disk-types"]
+= Boot disk types
+
+You can generate the following types of boot disks:
+
+ifndef::satellite[]
+Generic image::
+A boot ISO that is not associated with a specific host.
```

Comment:
It has the same description as a subnet image.
AFAIK, the only difference with the subnet image is that it doesn't support UEFI systems so that I would mention it.

Reply:
... and it cannot be generated for a specific smart proxy.